lib/evaluation/handlers/sameSubjectEvaluationHandler.ts

```diff
@@ -231,7 +211,7 @@ export class SameSubjectEvaluationHandler extends AbstractEvaluationHandler {
     status: 'info' | 'warn' | 'error'
   ): (inputDescriptorIds: Set<string>, fieldIdSet: Set<string>) => void {
     return (inputDescriptorIds: Set<string>, fieldIdSet: Set<string>) => {
-      const credentialSubjectsSet: Set<string> = new Set<string>();
+      const credentialSubjectsSet: Set<unknown> = new Set<unknown>();
```

Comment:
I think this the result of removing the CredentialSubject, by an unknown, which I believe is an error as CredentialSubject should have an optional id and additional unknow key-values (similar to what it was, but extended a bit)